django/db/models/options.py

```diff
@@ -65,9 +65,9 @@ def make_immutable_fields_list(name, data):
 
 @python_2_unicode_compatible
 class Options(object):
-    FORWARD_PROPERTIES = ('fields', 'many_to_many', 'concrete_fields',
-                          'local_concrete_fields', '_forward_fields_map')
-    REVERSE_PROPERTIES = ('related_objects', 'fields_map', '_relation_tree')
+    FORWARD_PROPERTIES = {'fields', 'many_to_many', 'concrete_fields',
+                          'local_concrete_fields', '_forward_fields_map'}
+    REVERSE_PROPERTIES = {'related_objects', 'fields_map', '_relation_tree'}
```

Comment:
This is not a matter of performance. We care about a particular set of values, not about their particular order. Set is the idiomatic Python type to use here.